packages/documentation/docs/vuefire/binding-subscriptions.md

```diff
@@ -4,14 +4,14 @@ In Vuefire, subscriptions to changes are handled transparently, that's why we al
 
 There are two ways of binding a Reference to the Database with Vuefire:
 
-- Using the `firebase`/`firestore` option
-- Calling the injected methods `$bind`/`$rtdbBind`
+- Simple "declarative" method, which you can only use when you are always binding to the same source, for example, a data structure accessible to all users. This uses the `firebase` or `firestore` option inside `export default`.
+- More flexible "programmatic" binding, which you must use when personalising what you are binding to, for example, binding a "current user" variable to the relevant user entry in a database. This uses the injected method `$rtdbBind` or `$bind`: 
 
-Once a Reference is bound, Vuefire will keep the local version in sync with the remote database. However, this synchronisation **is only one-way**, the local state is always a reflection of the remote Database that you should treat as read only state. If you want to [push changes to the remote Database](./writing-data.md), you need to use Firebase JS SDK. But, more about that later.
+Once a Reference is bound, Vuefire will keep the local version updated in line with the remote database. However, this synchronisation **is only one-way**. Do not modify the variable, because (a) it will not change the remote Database and (b) it can be overwritten at any time by Vuefire. To [write changes to the Database](./writing-data.md), you must use the Firebase JS SDK.
```

Comment:
updated in line -> synchronized

modify the **local** variable
add (eg: `this.user.name = 'John'`)

Reply:
Great.

(I personally like to use "synchronize" for bidirectional stuff, and use "updated" for unidirectional, but it's only a personal preference and, after all, it is your *excellent* repo for which I am very grateful!)

Reply:
thanks, I'm glad it's useful to you! In this case I prefer the word synchronize as it means something happening at the same time